apps/web/src/pages/_app.tsx

```diff
@@ -45,7 +75,10 @@ const MyApp = ({ Component, pageProps }: Props) => {
 
   useEffect(() => {
     globalThis.document.documentElement.classList.add(
-      chalkboradSEFont.className
+      process.env.NODE_ENV === "production"
```

Comment:
i don't think we need this, just import all the different fonts and you can reference them directly using tailwind